src/main.rs

```diff
@@ -29,17 +30,8 @@ use crate::util::deserialize_file;
 use crate::widgets::text::TextWidget;
 use crate::widgets::{I3BarWidget, State};
 
-fn main() {
-    tokio::runtime::Builder::new_multi_thread()
-        .worker_threads(2)
-        .max_blocking_threads(2)
-        .enable_all()
-        .build()
-        .unwrap()
-        .block_on(_main());
-}
-
-pub async fn _main() {
+#[tokio::main(flavor = "current_thread")]
```

Comment:
Does it improve resource-efficiency?

Reply:
Not really but I don't think it is a good idea to let tokio start a bunch of threads if we don't use them, especially if we continue spawning blocks with `spawn_local`.

Reply:
Actually, it increases the number or threads:
### Before
![image](https://user-images.githubusercontent.com/34583604/114618167-ba14c580-9cb1-11eb-9078-980861955ecf.png)
### After
![image](https://user-images.githubusercontent.com/34583604/114618187-bed97980-9cb1-11eb-89e9-c94487cf5536.png)

Reply:
That's actually very weird, it actually works as I had expected when disabling feature `rt-multi-thread` for tokio in *Cargo.toml*.

![Screenshot_2021-04-14_19-25-44](https://user-images.githubusercontent.com/1173464/114752991-3c0afa00-9d57-11eb-9c72-a0e0d15214cd.png)

I suppose this is a bug so I'll try to reproduce it in a minimal context and file an issue :/

Should I still revert this change? I have a tendency to change everything even when it is not a big deal and I understand that it can be rather intrusive :|

Reply:
Does using my config helps reproducing the issue?
```toml
[[block]]
block = "github"
token = "..."
hide = true

[[block]]
block = "wifi"
format = "{signal}"
format_alt = "{ssid} {frequency}"

[[block]]
block = "net"
format = "{graph_down}{speed_down;K}{speed_up;K}"
format_alt = "{interface}"

[[block]]
block = "custom"
command = "du -hs / 2>/dev/null | tr -d /"
interval = 600

[[block]]
block = "cpu"
format = "{barchart} {utilization}"
format_alt = "{freq}"
interval = 1

[[block]]
block = "memory"
format_mem = "{mem_used_percents:2}"
clickable = false

[[block]]
block = "temperature"
chip = "*-isa-*"
format = "{avg}"

[[block]]
block = "music"

[[block]]
block = "sway_kbd"
[block.mappings]
"English (Workman)" = "EN"
"Russian" = "RU"

[[block]]
block = "time"
icons_format = " "
interval = 30
format = "%d/%m %R"
```

Reply:
I reproduce the issue with my own config but it is fixed by commenting out [this line](https://github.com/MaxVerevkin/swaystatus/blob/master/Cargo.toml#L36) in *Cargo.toml* (which is the setup in which my screenshot was taken). I don't think this is how tokio is intended to work though.

Reply:
Well, if removing that feature solves the issue then i quess it's fine.